docs_src/microservices/core/metadata/Purpose.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Core Metadata - Purpose
+---
+
+# Core Metadata - Purpose
+
+The core metadata micro service has manages the knowledge about [devices](../../../general/Definitions.md#device) and sensors. This information is used by other services (Device, Command, etc) to communicate with them.
```

Comment:
Sorry, missed this in my previous suggestion.
```suggestion
The core metadata micro service manages the knowledge about [devices](../../../general/Definitions.md#device) and sensors. This information is used by other services (Device, Command, etc) to communicate with the devices.
```

Reply:
fixed